### DISCUSSION TITLE
Can i hire someone to install SO on my laptop?

### DISCUSSION THREAD

**Discussion by iqworks** [Security-Onion-Solutions/securityonion#5610] [Unsupported Versions] [answered]
I am wondering how much something like this might cost and where i can i find this type of resource?

**Comment by dougburks** [comment, marked as answer]
We offer Professional Services, but it's primarily aimed at enterprise teams and not home users:
https://securityonionsolutions.com/support

**Comment by iqworks** [comment]
I get that, thanks

 

From: Doug Burks ***@***.***> 
Sent: Wednesday, September 22, 2021 9:57 AM
To: Security-Onion-Solutions/securityonion
***@***.***>
Cc: iqworks ***@***.***>; Author ***@***.***>
Subject: Re: [Security-Onion-Solutions/securityonion] Can i hire someone to
install SO on my laptop? (Discussion #5610)

 

We offer Professional Services, but it's primarily aimed at enterprise teams
and not home users:
https://securityonionsolutions.com/support

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/Security-Onion-Solutions/securityonion/discussions/5610#
discussioncomment-1370735> , or unsubscribe
<https://github.com/notifications/unsubscribe-auth/ABKPV3YOLVATCLADZUNEQ5LUD
IDFRANCNFSM5EPBXG5A> .
Triage notifications on the go with GitHub Mobile for iOS
<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&m
t=8&pt=524675>  or Android
<https://play.google.com/store/apps/details?id=com.github.android&referrer=u
tm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>
.
<https://github.com/notifications/beacon/ABKPV32HIYAA62ZQJ7HR7RDUDIDFRA5CNFS
M5EPBXG5KYY3PNVWWK3TUL52HS4DFWFCGS43DOVZXG2LPNZBW63LNMVXHJKTDN5WW2ZLOORPWSZG
OAAKOU3Y.gif> 



-- 
This email has been checked for viruses by AVG.
https://www.avg.com
